vnext/Microsoft.ReactNative/IViewManager.idl

```diff
@@ -46,4 +46,10 @@ namespace Microsoft.ReactNative
 
     void ReplaceChild(XAML_NAMESPACE.FrameworkElement parent, XAML_NAMESPACE.UIElement oldChild, XAML_NAMESPACE.UIElement newChild);
   }
+
+  [webhosthidden]
+  interface IViewManagerRequiresNativeLayout
```

Comment:
consider adding a DOC_STRING("Implement this interface when your view manager is responsible for its own layout (i.e. it will self-size on its own") or something like that